partials/loop.hbs

```diff
@@ -13,7 +13,7 @@
         <p>{{excerpt words="26"}} <a class="read-more" href="{{url}}">&raquo;</a></p>
     </section>
     <footer class="post-meta">
-        {{#if author.image}}<img class="author-thumb" src="{{author.image}}" alt="{{author.name}}" nopin="nopin" />{{/if}}
+        {{#if author.image}}<a class="author-thumb" style="background-image: url({{author.image}})" nopin="nopin" />{{/if}}
```

Comment:
This is invalid HTML because `<a>` elements aren't void-elements and so can't be self-closing.

@JohnONolan you mentioned in https://github.com/TryGhost/Casper/issues/237#issuecomment-179026938 that it needs to be an `a` tag, could you elaborate on the reasoning?